packages/web-app/src/modules/reward-views/components/RewardItem.tsx

```diff
@@ -137,12 +139,12 @@ class _RewardItem extends Component<Props> {
         <div className={classes.textContainer}>
           <div className={classes.nameText}>{reward ? reward.name : <Skeleton />}</div>
           <div className={classes.subTextContainer}>
-            {reward && reward.originalPrice && !outOfStock && (
+            {shouldShowDiscount && (
               <div className={classnames(classes.discountLabel)}>
                 {getPercentOff(reward.originalPrice, reward.price)}{' '}
               </div>
             )}
-            {reward && reward.originalPrice && !outOfStock ? (
+            {shouldShowDiscount ? (
               <div className={classes.priceText}>
                 <span className={classes.originalPrice}>{reward.originalPrice}</span> ${reward.price}
```

Comment:
```suggestion
                <span className={classes.originalPrice}>${reward.originalPrice}</span> ${reward.price}
```
it looks like we're missing a dollar sign on the discount price. this should fix it (unless that was intentional). 
![image](https://user-images.githubusercontent.com/37646831/201136416-1e87bb1f-dff1-40fe-a04c-bd39691e52ab.png)

it also looks like the discount tag spans over 2 lines when we have 2 lines of prices. i don't think this is desired behavior. can you please double check with armin/ivan? if they say it's okay, i'll go ahead and approve.

thank you!

Reply:
it looks like we're also going to need this in the `alpha` branch. shoot me a message if you have any questions and happy to talk through it with you.

Reply:
@jpark799 thanks for that, fixed "dollar sign" and one-line discount label
created separate PR for alpha - https://github.com/SaladTechnologies/salad-applications/pull/963